src/main/java/seedu/duke/commands/TodoCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Task;
+import seedu.duke.trackables.Todo;
+
+import java.util.List;
+
+public class TodoCommand extends Command {
```

Comment:
Maybe it's good to rename the TodoCommand, DeadlineCommand and EventCommand to maybe AddToDoCommand, AddDeadlineCommand and AddEventCommand? This can improve the clarity of the code.